desktop/src/main/java/org/destinationsol/desktop/SolDesktop.java

```diff
@@ -120,7 +199,8 @@ public Path create(String fileName, List<String> lines) {
                     lines.add(line);
                 }
                 br.close();
-            } catch (IOException ignore) { }
+            } catch (IOException ignore) {
```

Comment:
Any scenario in which we might want to log this to `System.out.println` or something? Sometimes the old codebase seemingly just ignores stuff because easy. Or is this a place in which this could never be useful?

Reply:
Reads all the lines in file and returns them in List. When no file exists or can not be read, there are no lines to be read, and thus, no lines, or empty list, should be returned. Logic seems fine to me.